build.gradle.kts

```diff
@@ -39,12 +39,28 @@ allprojects {
       url = uri("https://api.mapbox.com/downloads/v2/releases/maven")
       credentials {
         username = "mapbox"
-        password = System.getenv("SDK_REGISTRY_TOKEN") ?: project.property("SDK_REGISTRY_TOKEN") as String
+        password =
+          System.getenv("SDK_REGISTRY_TOKEN") ?: project.property("SDK_REGISTRY_TOKEN") as String
       }
       authentication {
         create<BasicAuthentication>("basic")
       }
     }
+    val releaseTagRegex = "^v[0-9]+\\.[0-9]+\\.[0-9]+.*\$".toRegex()
+    val buildingRelease = releaseTagRegex.matches(System.getenv("CIRCLE_TAG") ?: "")
```

Comment:
nit: shall we move this `buildingRelease` detection to a separate function for reuse?

Reply:
I moved it to Project.kt